docs/topics/installing-cli-tool.adoc

```diff
@@ -20,7 +20,22 @@ However, if you want to analyze applications in languages other than Java or, fo
 
 .Prerequisites
 
-* Red Hat Container Registry Authentication for `registry.redhat.io`. Red Hat distributes container images from `registry.redhat.io`, which requires authentication. For more details, see link:https://access.redhat.com/RegistryAuthentication[Red Hat Container Registry Authentication].
+The following are the prerequisites for the {ProductFullName} CLI installation:
+
+* Java Development Kit (JDK) is installed. {ProductShortName} supports the following JDKs:
+
+** OpenJDK 17 or later
+** Oracle JDK 17 or later
+** Eclipse Temurin™ JDK 17 or later
```

Comment:
This is required if user wants to run in **local** mode and is not required if user will run in container mode (legacy java or any other language)